six/modules/c++/six.sidd/source/Utilities.cpp

```diff
@@ -418,6 +418,8 @@ six::PolarizationType _convertDualPolarization(six::DualPolarizationType pol,
                         : six::PolarizationType::RHC;
     case six::DualPolarizationType::LHC_LHC:
         return six::PolarizationType::LHC;
+    case six::DualPolarizationType::UNKNOWN:

```

Comment:
I think line endings are goofed up on these two lines (don't just do a dos2unix under this pull request as lots of lines in this file unfortunately are wrong)